Parking/OnStreetParking/doc/spec.md

```diff
@@ -152,6 +165,18 @@ This value must aggregate free spots from all groups devoted to special parking
     + Default unit: Meters
     + Optional
 
++ `maximumAllowedHeight` : Maximum allowed height for vehicles. If there are multiple zones, it will be the minimum height of
+all the zones.
+    + Attribute type: [Number](http://schema.org/Number)
+    + Default unit: Meters
+    + Optional
+
```

Comment:
I have my doubts that this property make sense on `OnstreetParking`, can you point me to an example? Are you sure that when a height restriction is present you are talking about an off street parking in the outside?